packages/core/src/@types/configs.ts

```diff
@@ -91,3 +91,16 @@ export interface IZapConfigArgs extends IOrderConfigArgs {
 export interface IWithdrawConfigArgs extends IOrderConfigArgs {
   suppliedLPAsset: AssetAmount<IAssetAmountMetadata>;
 }
+
+/**
+ * Interface describing the method arguments for creating a pool
+ * in the V3 Pool Contract.
+ */
+export interface IMintV3PoolConfigArgs extends IBaseConfig {
+  assetA: AssetAmount<IAssetAmountMetadata>;
+  assetB: AssetAmount<IAssetAmountMetadata>;
+  fees: TFee;
```

Comment:
How would we pass the decaying fees here?

I get that when passing sth. like `[5n, 10n]`, the fee would be `0.05%`, correct?

But for v3 pools, we planned to add the `start` and `end` fee as well as the `slot` up to which the fees will then decrease. How would we set that up?

Reply:
Oh i saw in another file that we would pass sth. like `feeDecayEnd` - shouldn't that be in the config as well?

Reply:
`[5n, 10n]` means "start at 0.05% as of marketTimings[0], and increase to 0.1% by marketTimings[1]"

Reply:
Okay, and what happens if we pass something like:

`fees: [5n, 10n]` and `marketTimings: [Date.now()]`

This would imply we have a dynamic fee, but a static `marketTiming` - so the pool would much likely have a static fee of `0.05%`, correct?

And out of curiosity, what happens if the second value of the `marketTiming` array is smaller than the first one? We wouldn't allow that from our UI, but it's theoretically possible. Do we handle that case in the SDK?

Reply:
I think that should fail, becuase it wouldn't have a value for the feeFinal time; In practice, We're not implementing the fees right now, so just pass the same values for both.

As for what would happen if you try to build a tx with funky values like a feeFinal before marketOpen, the contract will reject most strange cases.

If you specify [5n, 10n] and [Date.now(), Date.now()], then it will just start at 0.1%.

Reply:
Those cases for failing are good, I will add to the config validation.

Reply:
Added checks and tests here: https://github.com/SundaeSwap-finance/sundae-sdk/pull/29/commits/13f86319d2e0f04e13449cdfb94c2ec4faa1541b